htmlbasic/bootstrap2.html/homework2-1.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
```

Comment:
Still many errors when using the validator (19). If you are encountering problems you are not sure how to fix, please contact me on Slack